schema.sql

```diff
@@ -19,6 +19,20 @@ create table worksheets (
   unique(id)
 );
 
+drop table if exists worksheet_edits;
+create table worksheet_edits (
+  edit_id        uuid,
+  created_at     bigint,
```

Comment:
on the one hand is nice to get this level of precision, but on the other hand do we want to lose all the sql functionality that is possible using a timestamp?

postgres resolves to microsecond. maybe that's enough?

https://www.postgresql.org/docs/current/static/datatype-datetime.html

Reply:
After reviewing, recommend we keep as is:
- The `timestamp` type of postgres still deals with time zones in a confusing way, whereas this approach is bullet proof (because unix epoch a time zone less concept)
- Even with `timestamp without time zone`, you end up with a confusing interaction because the rest of the system is time zone aware (e.g. `now()` will convert in the local timezone, so if you were to make queries such as `select * from worksheet_edits where created_at > now() - interval 24 hour` you would have incorrect results depending on local settings)
- Typically, there is very little data manipulation done directly at SQL level. Especially with worksheets, you'd expect a programmatic way to query edits, and therefore we can resolve in the library all time related concerns.